targets/ipq40xx

```diff
@@ -0,0 +1,38 @@
+ATH10K_PACKAGES_IPQ40XX=
+if [ "$GLUON_WLAN_MESH" = 'ibss' ]; then
+	ATH10K_PACKAGES_IPQ40XX='-kmod-ath10k kmod-ath10k-ct -ath10k-firmware-qca4019 ath10k-firmware-qca4019-ct'
+fi
+
+# AVM
+
+device avm-fritz-box-4040 avm_fritzbox-4040
+factory
+extra_image '-squashfs-eva' '-bootloader' '.bin'
```

Comment:
Hmm, we don't quote the arguments in other (factory_image etc.) definitions either, let's remove the quotation marks here as well.

Reply:
Woops, copy-paste from you commit message, will change.